org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "org.js.nuclear.Nuclear",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "command": "run.sh",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host"
```

Comment:
What does it need host filesystem access for?

Reply:
It's for downloads of songs. You can set the download location and the locations of where songs are read from